accepted/PSR-4-autoloader.md

```diff
@@ -70,6 +70,6 @@ For example implementations of autoloaders conforming to the specification,
 please see the [examples file][]. Example implementations MUST NOT be regarded
 as part of the specification and MAY change at any time.
 
-[autoloading]: http://php.net/autoload
+[autoloading]: php.net/manual/en/language.oop5.autoload.php
```

Comment:
```suggestion
[autoloading]: https://php.net/manual/en/language.oop5.autoload.php
```

We do want the protocol in there, for consistency.

Reply:
Thank you, done.